main.go

```diff
@@ -40,7 +40,9 @@ func ListRangeHandler(rw http.ResponseWriter, req *http.Request) {
 }
 
 func ListPushHandler(rw http.ResponseWriter, req *http.Request) {
-	key := mux.Vars(req)["key"]
+	key   :=  mux.Vars(req) ["key"]
+
+	/* testing block comment */
```

Comment:
Go provides C-style /* */ block comments and C++-style // line comments. Line comments are the norm; block comments appear mostly as package comments, but are useful within an expression or to disable large swaths of code.

📄 [Commentary](https://app.semasoftware.com/guides/613b974694340b2a26f110e4/commentary-540)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix